hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/AbstractContractRootDirectoryTest.java

```diff
@@ -197,7 +197,7 @@ public void testListEmptyRootDirectory() throws IOException {
     }
     FileStatus[] rootListStatus = fs.listStatus(root);
     assertEquals("listStatus on empty root-directory returned found: "
-        + join("\n", rootListStatus),
+        + join(rootListStatus, "\n"),
         0, rootListStatus.length);
```

Comment:
Can we do this? rather than normal joins
```
    Assertions.assertThat(rootListStatus).hasSize(0);
```

It does give similar trace and looks better :)

```
java.lang.AssertionError: 
Expected size:<0> but was:<1> in:
<[HdfsLocatedFileStatus{path=hdfs://localhost:51193/test; isDirectory=true; modification_time=1682104954707; access_time=0; owner=ayushsaxena; group=supergroup; permission=rwxr-xr-x; isSymlink=false; hasAcl=false; isEncrypted=false; isErasureCoded=false}]>

```

Reply:
Guess I had some race condition with Steve. Use the one which he suggested.